okio.astub

```diff
@@ -0,0 +1,22 @@
+import org.checkerframework.checker.mustcall.qual.*;
+import org.checkerframework.checker.calledmethods.qual.*;
+import org.checkerframework.dataflow.qual.SideEffectFree;
+import org.checkerframework.common.returnsreceiver.qual.This;
+
+package okio;
+class Buffer{
+    public @MustCallAlias BufferedSource buffer(@MustCallAlias Buffer this);
+    public @MustCallAlias BufferedSource buffer(@MustCallAlias Object this);
+}
+
+class ForwardingSource{
+    public @MustCallAlias ForwardingSource(@MustCallAlias Source delegate);
+}
+
+
+interface BufferedSource{
+    
+    // Long readall();
+}
```

Comment:
Since everything in this class is commented out, we can just remove it. This also applies to unused imports in stub files, of which there are quite a few.